content/en/blog/baremetal_performance/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "How to achieve baremetal performance inside your virtual machine"
+date: 2022-05-19T08:00:00Z
+draft: false
+contributors: [Cloud Hypervisor Team]
+---
+
+There are use cases for which a workload needs to access specific hardware such
+as accelerators, GPU or network adapters to reach high performance. And since
+these workloads run inside virtual machines (VM) for security reasons, the
+challenge is to make this hardware available from within the VM, but without
+degrading the performance that can be achieved from baremetal.
+
+### VFIO, the first choice
+
+This is a framework that has been around for quite some time now, allowing a
+PCI device to be bound to the `vfio-pci` driver instead of the default driver
+it is usually attached to.
+
+It exposes an ioctl interface so that a userspace program can interact with the
+device and retrieve all the information it needs. In the virtualization case,
+this userspace program is the Virtual Machine Monitor (VMM), Cloud Hypervisor
+for instance, which after it got all the information from the device, exposes
+them to the guest. And that is how the workload can access the device inside the
+VM, exactly the same way it would if it was directly running on baremetal.
+
+Now that we've covered the functional aspect, let's have a look at the way we
+can achieve native performance from the guest.

```

Comment:
You're using native here instead of baremetal. Would be best to stick with one piece of terminology.